app/controllers/general.php

```diff
@@ -36,6 +38,26 @@
     /** @var Utopia\Locale\Locale $locale */
     /** @var array $clients */
 
+    /*
+     * Request format
+    */
+    $route = $utopia->match($request);
+    Request::setRoute($route);
+
+    $requestFormat = $request->getHeader('x-appwrite-response-format', App::getEnv('_APP_SYSTEM_RESPONSE_FORMAT', ''));
+    if ($requestFormat) {
+        switch($requestFormat) {
+            // TODO: For some reason console is still on 0.11. We dont want this filter logic in console, console uses 0.12 SDK
```

Comment:
@TorstenDittmann didn't we update the console SDK?

Reply:
Had chat with torsten, all good now.